tests/server/test_server.py

```diff
@@ -376,11 +372,14 @@ async def test_log_level_param():
     )
 
     # Check there are no debug logs
-    output = await sync_query(q, 'INFO')
+    response = await client.post("/query", json = q.dict())
+    output = response.json()
     assert not any(l['level'] == 'DEBUG' for l in output['logs'])
 
     # Check there are now debug logs
-    output = await sync_query(q, 'DEBUG')
+    response = await client.post("/query", json = q.dict(),
+                                 params = {"log_level" : "DEBUG"})
```

Comment:
```suggestion
                                 params={"log_level": "DEBUG"})
```
Please use a PEP8 linter to catch these whitespace inconsistencies.

Reply:
This applies in many other places here as well.